src/streams/streamsReducer.js

```diff
@@ -37,14 +68,12 @@ export default (
               );
 
             case 'update':
-              return state.map(stream =>
-                stream.stream_id === event.stream_id
-                  ? {
-                      ...stream,
-                      [event.property]: event.value,
-                    }
-                  : stream,
-              );
+              return state.map(stream => {
```

Comment:
Could convert this to a ternary operator.

Reply:
Yeah. I don't really have much of a preference on this one.